deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -605,6 +605,8 @@ cephObjectStores:
         # sslCertificateRef:
         instances: 1
         priorityClassName: system-cluster-critical
+        logging:
+          opsLogEnabled: false
```

Comment:
As a note, it seems like we aren't allowing this to be enabled from Helm. I don't have an issue with that, but I do want to point it out for others to comment on.

Reply:
This would just be the default. Users should be able to override any of the settings in the values.yaml.

Reply:
Oh, right. I forgot the whole CephObjectStore was expressed in the chart! I'll mark this resolved so it's not confusing